components/ProfileDrawer/Name.jsx

```diff
@@ -1,16 +1,16 @@
 import { useState } from 'react';
-import { getStringNoLocale } from '@inrupt/solid-client/thing/get'
-import { setStringNoLocale } from '@inrupt/solid-client/thing/set'
+import { getStringNoLocale } from '@inrupt/solid-client/thing/get';
```

Comment:
So, I think we agreed to use Prettier to format things on save, and specifically the Prettier VSCode extension. I re-enabled it for this, so anytime I edited a file that hadn't been reformatted by Prettier before it would reformat stuff. I tired to isolate things, especially if it was a one line change in a file, but definitely didn't do it everywhere.

It looks like we can change Prpettier settings to remove semi-colons: https://github.com/prettier/prettier-vscode#prettiersemi-default-true. We had talked about doing on pass to reformat everything according to the new rules. I think if I do that on main and then rebase on that branch, it should remove all the noise in this PR. Does that sound good?